tests/Table/CaptionTest.php

```
@@ -0,0 +1,863 @@
+                )
+                ->render(),
+            "Failed asserting that element renders correctly with 'events()' method.",
+        );
+    }
+
+    public function testRenderWithGlobalDefaultsAreApplied(): void
+    {
+        SimpleFactory::setDefaults(
+            Caption::class,
+            ['class' => 'default-class'],
+        );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Core\Factory\SimpleFactory' in method 'testRenderWithGlobalDefaultsAreApplied'. (undefined)

(StaticAccess)